pkgs/build-support/test-utilities.nix

```
@@ -0,0 +1,8 @@
+{ runCommand }:
+
+{
+    testVersion = command: args: version: runCommand "test-version" { meta.timeout = 3; } ''
```

```suggestion
    testVersion = command: args: version: runCommand "test-version" { meta.timeout = 60; } ''
```

The need for a timeout should be exceedingly rare, whereas load can be unpredictable. Let's avoid false negatives, because those come at the much greater cost: getting a human involved.

---

Also, some programs take an inappropriate amount of time to fully initialize, even if they only show the help or version text.
This happens with JVM apps, but I've seen other programs start slow for `--help`.